x-pack/test/cases_api_integration/security_and_spaces/tests/common/user_actions/find_user_actions.ts

```diff
@@ -0,0 +1,935 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import {
+  ActionTypes,
+  CaseResponse,
+  CaseSeverity,
+  CaseStatuses,
+  CommentUserAction,
+  ConnectorTypes,
+  FindTypes,
+} from '@kbn/cases-plugin/common/api';
+import {
+  globalRead,
+  noKibanaPrivileges,
+  obsOnly,
+  obsOnlyRead,
+  secOnly,
+  secOnlyRead,
+  superUser,
+} from '../../../../common/lib/authentication/users';
+import { findCaseUserActions } from '../../../../common/lib/user_actions';
+import {
+  getPostCaseRequest,
+  persistableStateAttachment,
+  postCommentActionsReq,
+  postCommentAlertReq,
+  postCommentUserReq,
+  postExternalReferenceESReq,
+} from '../../../../common/lib/mock';
+import {
+  deleteAllCaseItems,
+  createCase,
+  updateCase,
+  createComment,
+} from '../../../../common/lib/utils';
+
+import { FtrProviderContext } from '../../../../common/ftr_provider_context';
+
+// eslint-disable-next-line import/no-default-export
+export default ({ getService }: FtrProviderContext): void => {
+  const supertest = getService('supertest');
+  const es = getService('es');
+
+  describe('find_user_actions', () => {
+    afterEach(async () => {
+      await deleteAllCaseItems(es);
+    });
+
+    describe('sorting', () => {
```

Comment:
Maybe we can add a test with no parameters (check my comment about making the query params optional) to see if the defaults are as expected.